### ISSUE TITLE
initial implementation

### ISSUE THREAD

**Issue by av8ta** [AgregoreWeb/extension-agregore-render-json#1]
Hi @RangerMauve I've got a draft implementation at https://github.com/av8ta/extension-agregore-render-json 

Needs some styling work yet, and functionality like being able to copy and paste data, but I thought I'd throw it up now on github so you can have a look at it and see what you think :grin: 

It uses a simple virtual dom to build and mount elements. With a walk function to recursively walk the json data to build the virtual dom.

tailwindcss for styling because I'm utterly in love with that project and it's fun to pair it with [twind.dev](https://twind.dev/handbook/the-shim.html#basic-usage) shim so no nasty build steps are required!

**Comment by RangerMauve** [comment]
Sweet. Thank a gain for taking this on. Would you be down to do a pull request to this repo so that I could do a review via that? 😁

One thing that'd be good to double check is that the twind.dev shim will work offline using files that are fully local to the extension. 😁

**Comment by RangerMauve** [comment]
Also, I can't seem to get it to run when I clone it into my extensions folder. 😅 This is the URL I tested: https://blog.mauve.moe/posts/posts.json

**Comment by av8ta** [comment]
> Also, I can't seem to get it to run when I clone it into my extensions folder. sweat_smile This is the URL I tested: https://blog.mauve.moe/posts/posts.json

I'm getting `504 Gateway Time-out` in chrome so maybe you have an issue with your server :sweat_smile:

**Comment by av8ta** [comment]
> One thing that'd be good to double check is that the twind.dev shim will work offline using files that are fully local to the extension. grin

Absolutely, must be local-first! What we have now is just a quick and dirty solution to talk about and decide where to take it :grimacing: I need to include your theme css file too so themes can work correctly.

> Sweet. Thank a gain for taking this on. Would you be down to do a pull request to this repo so that I could do a review via that? grin

Sure, I actually tried pushing here initially, do I have permissions to push to this repo? But, for sure pull requests are a better idea anyway.

**Comment by RangerMauve** [comment]
Whoops, forgot to add this repo to the GH team. You should be able to push now. 😅
